### PR TITLE
feat: overrule participants 

### PR DESCRIPTION
**Pull request by Bateyjosue**
## In this PR

- [x] Create action handler to overrule participants

### CODE REVIEW

File: srv/service/bonusTranche.service.cds [hunk 1]
```diff
@@ -22,4 +22,6 @@ service BonusTrancheService @(requires: 'authenticated-user') {
                               trancheWeight : Decimal,
                               Location_ID : String @mandatory,
                               targets : array of Target @mandatory ) returns BonusTranche;
+
+    action overruleParticipant(participants: array of UUID, justification: String) returns TrancheParticipation;
```
Comment (ishimwe07) [inline srv/service/bonusTranche.service.cds:26]: For the return value, If user send many tranche participations to overrule the amount, which single tranche participation will be returned exactly?

Comment (ishimwe07) [inline srv/service/bonusTranche.service.cds:26]: For The overrule, It is the admin overruling the final amount of a participants, how is the new final amount is getting to the backend?

File: src/handlers/overrule-participant.action.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    try {
+      for (const participant of participants) {
+        await UPDATE(TrancheParticipation.name)
+          .where({ ID: participant })
+          .with({
+            overRuled: true,
+            justification: justification
+          })
+      }
+      return {
```
Comment (ishimwe07) [inline src/handlers/overrule-participant.action.ts:30]: The return value should be as the one mentioned while declaring an action in the service file.

**Review by ishimwe07 — CHANGES_REQUESTED** [review]
# Kudos @Bateyjosue   implementing the feature
## Highlights
I left few comments
- The over rule of the final amount and how it is being updated
- The return type of the action

File: srv/service/bonusTranche.service.cds (hunk 1 above)
Comment (Bateyjosue) [reply srv/service/bonusTranche.service.cds:26]: Do you know the formula to calculate it??

**Review by ishimwe07 — APPROVED** [review]
# Nice work implementing the changes requested :tada:

**Review by pmy09 — APPROVED** [review]

**Pull request merged**